ibis/tests/expr/test_operations.py

```diff
@@ -188,27 +219,28 @@ def to_expr(self):
     assert isinstance(Log(arg=100).to_expr(), Logarithm)
 
 
-def test_operation_nodes_are_slotted(op):
-    assert hasattr(op, "__slots__")
-    assert not hasattr(op, "__dict__")
+def test_operation_nodes_are_slotted(operations):
+    for op in operations:
+        assert hasattr(op, "__slots__")
```

Comment:
Not entirely sure this test is worth keeping around.

Reply:
I'd rather keep it to prevent having `__dict__` in favor of `__slots__`